.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,51 @@
+name: release-e2e
+
+on:
+  release:
+    types: [created]
+  workflow_dispatch:
+
+jobs:
+  release-e2e-kind:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-go@v2
+      with:
+        go-version: '~1.18'
+
+    - name: install_kind_binary
+      env:
+        KIND_VERSION: v0.12.0
+        KIND_OS: kind-linux-amd64
+      run: |
+        curl -sLo kind "https://github.com/kubernetes-sigs/kind/releases/download/${KIND_VERSION}/${KIND_OS}"
+        chmod +x kind
+        sudo mv kind /bin/
+
+    - name: start_kind_cluster
+      run: |
+        # /bin/kind create cluster
+        make kind-cluster
```

Comment:
nit: this target deletes a cluster first before creating a new one with a specific name. In this CI context though, there is never going to be a pre-existing cluster. Does it make more sense to simply create a cluster here with the default name? 
Ala
```
kind create cluster
```

Reply:
That makes sense to me. I was being lazy and trying to use a pre-made target to get functionality for free but I didn't realize it was creating potential issues.